src/cleed++/LEED/Crystal.hh

```
@@ -0,0 +1,116 @@
+
+    /* getters */
+    double getRealOpticalPotential() const;
+    double getImaginaryOpticalPotential() const;
+    double getTemperature() const;
+
+    size_t getNumberOfRotationalSymmetries() const;
+    vector<double> getAxisOfRotationalSymmetry() const;
+
+    size_t getNumberOfMirrorPlanes() const;
+    const real *getMirrorPlaneGeometryPtr() const;
+    vector<double> &getMirrorPlaneGeometry() const;
```

Clang (14.0.6)

[warning] function 'getMirrorPlaneGeometry' should be marked [[nodiscard]]

(modernize-use-nodiscard)